README.md

```diff
@@ -1,147 +1,184 @@
 # LPython
 
-LPython is a Python compiler. It is in heavy development, currently in
-pre-alpha stage. Some of the goals of LPython:
-
-* The best possible performance for numerical, array-oriented code
-* Run on all platforms
-* Compile a subset of Python yet be fully compatible with Python
-* Explore designs so that LPython eventually can compile all Python code
-* Fast compilation
-* Excellent user-friendly diagnostic messages: error, warnings, hints, notes,
+LPython is a Python compiler. It is in heavy development, currently in pre-alpha stage. LPython works on Windows, macOS and Linux. Some of the goals of LPython include:
+
+- The best possible performance for numerical, array-oriented code
+- Run on all platforms
+- Compile a subset of Python yet be fully compatible with Python
+- Explore designs so that LPython eventually can compile all Python code
+- Fast compilation
+- Excellent user-friendly diagnostic messages: error, warnings, hints, notes,
   etc.
-* Ahead-of-Time compilation to binaries, plus interactive usage (Jupyter
-  notebook)
-* Transforming Python code to C++, Fortran and other languages
+- Ahead-of-Time compilation to binaries, plus interactive usage (Jupyter notebook)
+- Transforming Python code to C++, Fortran and other languages
 
 And more.
 
 # Installation
 
-LPython works on Windows, macOS and Linux.
+## Step 0: Prerequisites
 
-## Install Conda
+Here is the list of requirements needed to build LPython:
+
+- Python (3.10+)
+- Conda
+- Miniforge Prompt
+- Visual Studio (with "Desktop Development with C++" workload)
```

Comment:
I think this two are specific to Windows.

Reply:
I will do some changes.